spec/lang/step/operators.md

```diff
@@ -70,7 +70,7 @@ impl<M: Memory> Machine<M> {
 impl<M: Memory> Machine<M> {
     #[specr::argmatch(operator)]
     fn eval_bin_op(
-        &self,
+        &mut self,
```

Comment:
These functions deliberately took `&self`, please don't change that.

You can make RawEq an intrinsic instead of a BinOp.